sale_order_weight/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sale Order Weight
+==============================
+
+This module adds products weight in report for sale.order
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
```

Comment:
Please add Usage section and update Bug Tracker section, see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst as reference.